scripts/parse-xml-cvrs.py

```diff
@@ -18,13 +26,37 @@ def findall(xml, tag):
     return xml.findall(tag, namespaces={"": ns})
 
 
+NUM_CAST = "# Number of Votes Cast in Contest"
+
+
 def get_directory_name(file_path):
     directory_path = os.path.dirname(file_path)
     directory_name = os.path.basename(directory_path)
     return directory_name
 
 
-def parse_cvr_file(file_path, use_directory_name_as_tabulator=False):
+def parse_scanned_ballot_file(file_path, cvr_workstation_mapping):
+    with open(file_path, "r", encoding="utf-8") as ballots_file:
+        headers, rows = read_ess_ballots_file(ballots_file)
```

Comment:
Oh interesting that this ESS ballots file parsing function works on this file! This file is technically not an ESS ballots file but a Hart scanned ballot information file. Looks like they're similar enough in structure that the same function works. But to avoid confusion, we should probably use this function instead:

https://github.com/votingworks/arlo/blob/b00d3f64ef627ee2ea9b6b63de8ec903fd99eab6/server/api/cvrs.py#L1079-L1081

Reply:
facepalm lol